packages/core/components/EmptyFileListMessage/FilterList.tsx

```diff
@@ -37,7 +37,10 @@ export default function FilterList(props: Props) {
         }
     }, [textRef, filters]);
 
-    const operator = filters.length > 1 ? "for values of" : "equal to";
+    const firstFilterValue = filters[0].value.toString();
+    let operator = "equal to";
+    if (filters.length > 1) operator = "for values of";
+    else if (firstFilterValue.includes("RANGE")) operator = "between";
```

Comment:
So if filters.length is <= 1, and filters[0] = RANGE, operator is "between"?  I think you don't want the 'else' here.
 if (filters.length > 1) {
      operator = "for values of";
    (firstFilterValue.includes("RANGE")) operator = "between";
}

Reply:
iirc, range is sent as a single string (e.g., `'RANGE(1,10)'` so the filter length in that case would be 1

Reply:
So 
if (filters.length == 1 && firstFilterValue.includes("RANGE")) operator = "between";
Thanks for the clarification